readme.md

```diff
@@ -90,7 +90,7 @@ To upgrade Neo-Express, enter the following command in a terminal window:
 Installing on Ubuntu 18.04 requires installing libsnappy-dev and libc6-dev via apt-get
 
 ``` shell
-> sudo apt install libsnappy-dev libc6-dev -y
+> sudo apt install libsnappy-dev libc6-dev librocksdb-dev -y
```

Comment:
Can we update the text above? Also, what versions of Ubuntu have you tested this on @metachris ?

Reply:
I tried this with the `dotnet/sdk:5.0` Docker image:

```
docker run --rm -it mcr.microsoft.com/dotnet/sdk:5.0

# Inside Docker
apt update
apt install libsnappy-dev libc6-dev librocksdb-dev -y
dotnet tool install -g Neo.Express --version 2.0.24-preview
export PATH="$PATH:/root/.dotnet/tools"

neoxp -v
neoxp -h
neoxp create
neoxp run
```